File: EducationApp.DataAccessLayer/Models/Filters/FilterPrintingEditionModel.cs
using EducationApp.DataAccessLayer.Entities.Enums;
using EducationApp.DataAccessLayer.Models.Filters.Base;
using System.Collections.Generic;

namespace EducationApp.DataAccessLayer.Models.Filters
{
    public class FilterPrintingEditionModel : BaseFilterModel
    {
        public string SearchString { get; set; }
        public Enums.Currency Currency { get; set; }
        public decimal PriceMinValue { get; set; }
        public decimal PriceMaxValue { get; set; }
        public IEnumerable<Enums.PrintingEditionType> PrintingEditionTypes { get; set; }
    }
}

File: AngularEducationStore/src/app/shared/models/filter/filter-printing-edition-model.ts
import { BaseFilterModel } from 'src/app/shared/models/filter/base/base-filter-model';
import { PageSize } from 'src/app/shared/enums/page-size';

export class FilterPrintingEditionModel extends BaseFilterModel {
    currency = 1;
    PriceMinValue = 0;
    PriceMaxValue = 10000;
    PrintingEditionTypes = new Array<number>();
    pageSize = PageSize.Six;
}
